### DISCUSSION TITLE
Can we use GPU for ffmpeg processing?

### DISCUSSION THREAD

**Discussion by Passaita** [nextcloud/recognize#1078] [General]
After a week of fights with the ESXi host being unstable with passtrough I finally have GPU working and processing pictures, now...
My CPU is still busy with ffmpeg decoding and I am wondering if there is a way to direct ffmpeg processing to GPU as it has fully-accelerated hardware-based video decoding and encoding for several popular codecs. With decoding/encoding offloaded, to the graphic card - the CPU can be used for other operations.
![image](https://github.com/nextcloud/recognize/assets/25120690/e7643ddd-ce7d-491b-9878-591feaa0aad9)

**Comment by marcelklehr** [comment]
See #655 where this has been discussed already
